src/github.com/getlantern/flashlight/android/bind.bash

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
```

Comment:
This file creates Java and Go bindings using gobind. See https://godoc.org/golang.org/x/mobile/cmd/gobind